.github/workflows/build.yml

```diff
@@ -26,9 +26,12 @@ jobs:
           cache: 'maven'
       - name: Build
         run: mvn clean install -T1C -Pjsdoc,minify --batch-mode --show-version
+      - name: Sonar Cloud - simulate (without sonar:sonar)
+        if: matrix.java == 21
+        run: mvn clean install -Pquick -pl !primefaces-integration-tests-jakarta
       - name: Sonar Cloud
         if: github.repository == 'primefaces/primefaces' && github.ref == 'refs/heads/master' && matrix.java == 21
-        run: mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Pquick -Dsonar.organization=primefaces -Dsonar.host.url=https://sonarcloud.io -Dsonar.token=${{secrets.SONAR_TOKEN}}  -fprimefaces/pom.xml
+        run: mvn clean install sonar:sonar -Pquick -pl !primefaces-integration-tests-jakarta -Dsonar.organization=primefaces -Dsonar.host.url=https://sonarcloud.io -Dsonar.token=${{secrets.SONAR_TOKEN}}
```

Comment:
change #1 - we run `mvn clean install sonar:sonar ...` for the whole project instead of only for Primefaces(-Core)
This is necessary to get code-coverage generated via "primefaces-integration-tests" included.